hazelcast/src/main/config-template/hazelcast-assembly.xml

```diff
@@ -16,11 +16,20 @@
   -->
 
 <!--
-  The default Hazelcast configuration.
+  The default Hazelcast @FILTER_DISTRO_NAME@ configuration.
 
   This file differs from hazelcast-default.xml in some settings
-  and it is used by the scripts in the ZIP/TAR distributions,
-  which are the basis for other distributions like docker & package managers.
+  and it is used by the scripts in the @FILTER_DISTRO_NAME@ distribution.
+
+  This configuration, compared to hazelcast-default.xml, enables more features.
+  It's focused on keeping balance between security and usability in the distribution.
+
+  To allow full external access in ZIP/TGZ distribution, follow these steps:
+  - set "hazelcast.socket.bind.any" property value to "true"
+  - set "enabled" attribute value in "network/interfaces" element to "false"
+  - in network/join configuration:
+    - set autodiscovery "enabled" attribute to "true"
+    - set tcp-ip "enabled" attribute to "false"
```

Comment:
This must be so horrible for a newcomer that I would rather delete it. Maybe we can put this in some FAQ section of the documentation. Opening everything to the world isn't something done very often.

Reply:
:+1: True, I'll remove the paragraph.